### PR TITLE
adding tip about responsive design and tables

### PR DESCRIPTION
**Pull request by brianelton**
Adding note from #526 regarding Responsive design. Covers #593

### CODE REVIEW

**Comment by brianelton** [comment]
Holding off on merging PR to see if we should add some more info regarding CSS grid layouts

**Comment by shawna-slh** [comment]
Preview: https://deploy-preview-718--wai-tutorials2.netlify.app/tutorials/tables/tips/

File: content/tables/tips.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ support: Developed by the Education and Outreach Working Group (<a href="https:/
 
 -   **Keep it simple:** Complex tables are more work for content creators as well as being harder to interpret for users. It’s usually better to [break up complex tables into simple individual tables](/tutorials/tables/multi-level/#split-up-multi-level-tables), each containing the data for one sub-topic.
 
+-   **Responsive design:** On smaller screens, responsive tables often change format. Ensure that the structural relationship is available in all formats.
```
Comment (shawna-slh) [inline content/tables/tips.md]: Would be good to include users with high zoom and/or large fonts. (I don't have a specific wording suggestion now though.)

Comment (brianelton) [reply content/tables/tips.md]: How about "On smaller screens, or when viewing content in a small viewport, responsive tables..." ?

Comment (brianelton) [reply content/tables/tips.md]: I have updated this.

**Comment by brianelton** [comment]
@shawna-slh any further comments before merging?

**Comment by shawna-slh** [comment]
>  "On smaller screens, or when viewing content in a small viewport, responsive tables..."

hum, this doesn't address my point. I have a large monitor and the viewport max -- so neither "On smaller screens, or when viewing content in a small viewport".

However, with large text and increased line spacing &/or browser zoom, I still get the "responsive" view.

**Comment by brianelton** [comment]
@shawna-slh how about "On smaller screens, or when viewing content in a small viewport, zoomed screen, or otherwise adjusted view, responsive tables often change format."

**Comment by shawna-slh** [comment]
how about: "Responsive design: On small screens, in a small viewport, in zoomed views, with enlarged text, and other situations, responsive tables often change format. Ensure that the structural relationship is available in all formats."
?

**Comment by brianelton** [comment]
@shawna-slh yes, that looks good. I have updated it.

**Comment by brianelton** [comment]
@shawna-slh please merge this PR

**Comment by shawna-slh** [comment]
Thanks Brian!
/me not able to make time to review. OK for @remibetin to merge and publish

**Pull request merged**